firestore/integration_test_internal/src/aggregate_query_test.cc

```diff
@@ -36,28 +30,155 @@ namespace {
 
 using AggregateQueryTest = FirestoreIntegrationTest;
 
-TEST_F(AggregateQueryTest, DefaultConstructor) {
+TEST_F(AggregateQueryTest, DefaultConstructorReturnsValidObject) {
```

Comment:
"ValidObject" is a bit confusing, because the test explicitly verifies that `is_valid()` returns false :)
I think in this case just naming it "default constructor" is fine because there is one, and only one, state to be verified.